File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // ask the user  for a number from 1 - 8
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int i = 0; i < height; i++)
    {
        // print out the spaces
        for (int j = height - 1; j > i; j--)
        {
            printf(" ");
        }
        // print out the triangle
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }

        // add the separator
        printf("  ");

        // print out the secondd triangle
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: python/mario-more/mario.py
# get the get_int function from cs50
from cs50 import get_int

# initialize height as 0
height = 0

# prompt the user to enter valid height
while height < 1 or height > 8:
    height = get_int("Height: ")

# use a loop to print out pyramid of given height
for i in range(1, height + 1):
    # print out spaces and hashes on the same line
    print(" " * (height - i) + "#" * i + "  " + "#" * i)
